code/modules/research/stock_parts.dm

```diff
@@ -44,6 +44,9 @@ If you create T5+ please take a pass at gene_modder.dm [L40]. Max_values MUST fi
 	name = "bluespace rapid part exchange device"
 	desc = "A version of the RPED that allows for replacement of parts and scanning from a distance, along with higher capacity for parts."
 	icon_state = "BS_RPED"
+	item_state = "BS_RPED"
+	lefthand_file = 'icons/mob/inhands/misc/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
```

Comment:
These are not needed , the inhand file for /obj/item/storage/part_replacer is already devices_left/righthand.dmi

Reply:
Got it! Misunderstood how that worked, thanks!